test/unit/specs/users/cd-redirect-to-login.spec.js

```diff
@@ -0,0 +1,22 @@
+import vueUnitHelper from 'vue-unit-helper';
+import RedirectToLoginComponent from '!!vue-loader?inject!@/users/cd-redirect-to-login';
+
+describe('Redirect to Login', () => {
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe('computed', () => {
+    const vm = vueUnitHelper(RedirectToLoginComponent({}));
```

Comment:
You should never have code directly in a `describe`. It'll execute immediately before any of the tests have run. Create the vm in the test itself, or in a `beforeEach`.

Reply:
I might have been lazy, agreed